src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -16306,22 +16306,22 @@ public class C4 : I2, I3
             var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
             comp.VerifyDiagnostics(
                 // (11,14): warning CS8645: 'I1<A>' is already listed in the interface list on type 'C1' with different nullability of reference types.
-                // public class C1 : I2, I1<A>
+                // public class C1 : I2, I1<A> 
                 Diagnostic(ErrorCode.WRN_DuplicateInterfaceWithNullabilityMismatchInBaseList, "C1").WithArguments("I1<A>", "C1").WithLocation(11, 14),
                 // (11,14): error CS8646: 'I1<A?>.M()' is explicitly implemented more than once.
-                // public class C1 : I2, I1<A>
+                // public class C1 : I2, I1<A> 
                 Diagnostic(ErrorCode.ERR_DuplicateExplicitImpl, "C1").WithArguments("I1<A?>.M()").WithLocation(11, 14),
                 // (11,14): error CS8646: 'I1<A>.M()' is explicitly implemented more than once.
-                // public class C1 : I2, I1<A>
+                // public class C1 : I2, I1<A> 
                 Diagnostic(ErrorCode.ERR_DuplicateExplicitImpl, "C1").WithArguments("I1<A>.M()").WithLocation(11, 14),
                 // (17,14): warning CS8645: 'I1<A?>' is already listed in the interface list on type 'C2' with different nullability of reference types.
-                // public class C2 : I1<A>, I2
+                // public class C2 : I1<A>, I2 
                 Diagnostic(ErrorCode.WRN_DuplicateInterfaceWithNullabilityMismatchInBaseList, "C2").WithArguments("I1<A?>", "C2").WithLocation(17, 14),
                 // (17,14): error CS8646: 'I1<A>.M()' is explicitly implemented more than once.
-                // public class C2 : I1<A>, I2
+                // public class C2 : I1<A>, I2 
                 Diagnostic(ErrorCode.ERR_DuplicateExplicitImpl, "C2").WithArguments("I1<A>.M()").WithLocation(17, 14),
                 // (17,14): error CS8646: 'I1<A?>.M()' is explicitly implemented more than once.
-                // public class C2 : I1<A>, I2
+                // public class C2 : I1<A>, I2 
                 Diagnostic(ErrorCode.ERR_DuplicateExplicitImpl, "C2").WithArguments("I1<A?>.M()").WithLocation(17, 14),
                 // (23,14): warning CS8645: 'I1<A?>' is already listed in the interface list on type 'C3' with different nullability of reference types.
```

Comment:
>// (23,14): warning CS8645: 'I1<A?>' is already listed in the interface list on type 'C3' with different nullability of reference types. [](start = 16, length = 136)

If we keep the new error, I would expect this warning to go away. Otherwise, it looks like we are reporting a warning and an error at the same time for the same problem